Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -246,6 +246,36 @@ double HGCalDDDConstants::cellSizeHex(int type) const {
   return cell;
 }
 
+int HGCalDDDConstants::cellType(int type, int cellU, int cellV) const {
+  // type=0: in the middle; 1..6: the edges clocwise from bottom left;
+  //     =11..16: the corners clockwise from bottom
+  int N = (type == 0) ? hgpar_->nCellsFine_ : hgpar_->nCellsCoarse_;
+  if (cellU == 0) {
+    if      (cellV == 0)         return 12;
```

Comment:
It would be clearer to make these cell type numbers into named constants (or an enum class) so we can tell what they mean at a glance

Reply:
Thought about it. Will do that